book_src/Operating System From 0 to 1.lyx

```diff
@@ -502,7 +502,7 @@ There are many books on this topic and even courses out there already, made
  book, and probably the students are assumed to obtain enough hardware knowledge
  beforehand from other courses.
  Or, the students only need to work within existing assignment framework:
- the only thing they need to do is filling the empty function body, run
+ the only thing they need to do is fill the empty function body, run

```

Comment:
I wish I could give you a more technical answer, but I'm honestly not sure of the exact reasons why "fill" is correct here. It's correct for the same reasons "run" and "complete" are correct in the same sentence though.